sphinx_immaterial/search_adapt.py

```diff
@@ -19,15 +19,22 @@
 class IndexBuilder(sphinx.search.IndexBuilder):
     def get_objects(
         self, fn2index: Dict[str, int]
-    ) -> Dict[str, Dict[str, Tuple[int, int, int, str, str]]]:
+    ) -> Dict[str, List[Tuple[int, int, int, str, str]]]:
         rv = super().get_objects(fn2index)
         onames = self._objnames
-        for prefix, children in rv.items():
+        for prefix in rv:
             if prefix:
                 name_prefix = prefix + "."
             else:
                 name_prefix = ""
-            for name, (docindex, typeindex, prio, shortanchor) in children.items():
+            if sphinx.version_info[0] >= 4 and sphinx.version_info[1] >= 3:
```

Comment:
Change this to `sphinx.version_info >= (4, 3)` as otherwise the condition will be false for e.g. version 5.3.

Reply:
Thanks that's much cleaner as well - changed in fac934c